### PR TITLE
Add JSTL sources jar for API project

### LINKED ISSUE #11718: JakartaEE: JSTL Add Source Jar
Add source jar to  io.openliberty.jakarta.jstl.2.0.

Useful for when creating overlays. 

This would be similar as to MyFaces:
https://github.com/OpenLiberty/open-liberty/blob/df18bd737d4a9a0803e66e89469c28ae50533460/dev/com.ibm.ws.org.apache.myfaces.2.3/build.gradle#L20

Current Jakarta JSTL version is 2.0.0-RC1. Maybe we'll wait for 2.0.0 release?

### PR DESCRIPTION
**Pull request by isaacrivriv**
fixes #11718

### CODE REVIEW

**Comment by isaacrivriv** [comment]
#build

**Review by volosied — APPROVED** [review]
Looks good. Thanks!

 I can the sources for the JSTL Impl in my PR.

File: dev/io.openliberty.jakarta.jstl.2.0/build.gradle [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
```
[the remark this answers is not in the document]
Comment (volosied) [reply dev/io.openliberty.jakarta.jstl.2.0/build.gradle:3]: lol

[the remark this answers is not in the document]
Comment (isaacrivriv) [reply dev/io.openliberty.jakarta.jstl.2.0/build.gradle:3]: Happy new year indeed lol Ill make the change!

**Comment by isaacrivriv** [comment]
#build

**Review by pnicolucci — APPROVED** [review]
Looks good, thanks!

**Comment by pnicolucci** [comment]
#run-libby-bot

**Pull request merged**